src/chrome/content/rules/Eater.xml

```diff
@@ -24,15 +24,13 @@
 		- Fonts on www from fonts.voxmedia.com
 
 -->
-<ruleset name="Eater.com (partial)">
+<ruleset name="Eater.com">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="www.eater.com" />
-
-	<!--	Complications:
-				-->
 	<target host="eater.com" />
+	<target host="*.eater.com" />
```

Comment:
Please have a look at the comments on the top. You may update/remove them if all hosts are working now.